.travis.yml

```diff
@@ -0,0 +1,13 @@
+matrix:
+  include:
+    - language: python
+      python: 3.7
+      install:
+        - pip install -r requirements.txt
+      script:
+        - python -m unittest
+
+    - language: node_js
+      node_js: "node"
```

Comment:
@blakes24 I suspect that the build could be failing due to the node_js version specified.

Reply:
@blakes24 do you still need more assistance on this pull request ? If not kindly update the badges.